src/messages/formatters.ts

```diff
@@ -145,6 +146,33 @@ export function hyperlink(content: string, url: string | URL, title?: string) {
 	return title ? `[${content}](${url} "${title}")` : `[${content}](${url})`;
 }
 
+/**
+ * Formats the user ID into a user mention.
+ * @param userId The user ID to format.
+ * @returns The formatted user mention.
+ */
```

Comment:
No need to specify the return iirc
```suggestion
/**
 * Formats the user ID into a user mention.
 * @param userId The user ID to format.
 */
```

Reply:
Everything else does.